adafruit_cursorcontrol/cursorcontrol_cursormanager.py

```diff
@@ -174,3 +161,42 @@ def _check_cursor_movement(self, pressed=None):
                 self._cursor.y -= self._cursor.speed
         else:
             raise AttributeError('Board must have a D-Pad or Joystick for use with CursorManager!')
+
+
+class DebouncedCursorManager(CursorManager):
+    """Simple interaction user interface interaction for Adafruit_CursorControl.
```

Comment:
Could you explain the purpose of `DebouncedCursorManager` here? 

This is how you mentioned the `DebouncedCursorManager` works to me: 
> The one with the debouncer would provide the additional press, release, is_pressed functionallity